conf/nginx.conf

```diff
@@ -8,9 +9,8 @@ location __PATH__/ {
 }
 
 location ~ __PATH__/\.(css|js|png|jpe?g|svg|eot|woff2?)$ {
-	more_set_headers Cache-Control 'max-age=2592000, must-revalidate, public';
+	more_set_headers 'Cache-Control: max-age=2592000, must-revalidate, public';
```

Comment:
Nice catch, I'll put it in another commit as this is not just lint but probably a fix!